### PR TITLE
update travis.yml to post to slack

### PR DESCRIPTION
**Pull request by superseacat**
reconf travis to post to slack

# Description

# Testing

- [ ] I created new unit tests
- [ ] All unit tests pass
- [x] I have tested manually

# Have you updated the README or other relevant documentation?

- [ ] Yes
- [ ] Not yet. I will do it next.
- [x] Not relevant

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,25 +1,27 @@
...
+- provider: script
+  script: env INTELLIJ_PUBLISH_CHANNEL=test ./gradlew publishPlugin
+  on:
+    branch: test
 notifications:
-  slack: $SLACK
\ No newline at end of file
+  slack:
+    rooms:
+      secure: 3dvnGSeKK+gZZPz6cQuA8EQZlRskdzGy2/FHcPvx21cafnhiDmEyxLiT60/GR/kmyTfRODhsgaPBD2pRyXlzOowBfyVWTHiPzmU412Q9kSAR5TL3M8O0ENlgZJs4Ujz9BB4R48aXBtyeUhYLj0fZyygDI4/UtJjsygveqUnAFS0I7djzLnpIIkIesB559xMwNjpooPxw4nUWARtRBK6jbXtRO8bFc/VprG4kV63dwFgr6TOF2cFcSQuQCbzu692QcBvXQKLqkFqCuJVxj0sYZrQfy+Ccbx4q+a4Db6qJ0pOu/fDIbv6XIh/GeILDOF8fIdj1kA+pWKTWn0xj3MVy/YBC5kZiVOlNVDuM+zZYQr8SfQzA5LVxEf/j6HoY3vk4U0e3xhg1bmFkUZHFEv5SmHutmam1qVJlEwVytFCqIizIU/yDeG9VE2vr2dIDceSt7bjBi48XrhnXSAvBolz5MXZFY/Cf/nnIfkCtCernf70unmVBnZ/HalNEi/AcNFnWX1LHB5et1KQ2LACstQALIWBTyHlj7x/Tzs2U4cVbULlwKZMmAGhdtvC1SysthARWwXNXg+dz4Ep65GCbcJAUAhzJy0xBy05tZbnOIJMw7m6MhNoo7E8gDZxpUFZbfO9nHqZ7OR7JIL7kUHmIYyl77XDeJPcoFi7YmmE53D/aJuM=
```
Comment (OlliKiljunen) [inline .travis.yml]: Shouldn't this be secure travis parameter?

Comment (superseacat) [reply .travis.yml]: this is now encrypted with travis cli (and [includes the workspace & token](https://docs.travis-ci.com/user/notifications/#configuring-slack-notifications))

Comment (superseacat) [reply .travis.yml]: fixin'

**Comment by superseacat** [comment]
I'll close this PR and create a new one so the checks would run again

**Pull request closed without merging**